frontend/src/bundles/common/components/header/header.tsx

```diff
@@ -0,0 +1,30 @@
+import logo from '~/assets/img/logo.svg';
+
+import { getValidClassNames } from '../../helpers/helpers.js';
+import { Layout, Link } from '../components.js';
+import { Message, Navigation } from './components/components.js';
+import styles from './styles.module.css';
+
+const Header = (): JSX.Element => {
+    return (
+        <header className={styles['header']}>
+            <Layout className={getValidClassNames(styles['header-container'])}>
```

Comment:
Why `getValidClassNames` is needed here? As far as I see you also pass below in the code single class using `styles[some-class]` but without this helper function

Reply:
Thank you for noticing. I'll fix this.